### PR TITLE
Pysmurf controller updates from p10r1

### PR DESCRIPTION
**Pull request by jlashner**
Bugfixes, better feeds and session data, and a few extra tasks from P10R1 testing .

## Description
This PR includes several additions that we made during P10R1, including:
- addition of the `overbias_tes` and `all_off` tasks
- Better session-data for bias_steps, including total dets in Rfrac range, and dets per bg in Rfrac range
- bias_steps publishes Rtes, Rfrac, and Si quantiles to grafana

## Motivation and Context
This was used in P10R1

## How Has This Been Tested?
P10R1

## Types of changes

- [X] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [X] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] Unless I am preparing a release, I have opened this PR onto the `develop` branch.

### CODE REVIEW

**Comment by BrianJKoopman** [comment]
Can you rebase on the latest `develop` and force push back here? There's a merge conflict that relates to the restructuring of the repo.

File: socs/agents/pysmurf_controller/agent.py [hunk 1]
```diff
@@ -706,12 +717,83 @@ def take_bias_steps(self, session, params):
...
+        -------
+        bgs : List[int]
+            List of bias groups to overbias. If this is set to None, it will
+            use all active bgs.
+        kwargs : dict
+            Additional kwargs to pass to the ``overbias_tes_all`` function.
+        """
+        kw = {'bias_groups': params['bgs']}
+        if params['kwargs'] is None:
+            kw.update(params['kwargs'])
```
Comment (BrianJKoopman) [inline socs/agents/pysmurf_controller/agent.py:785]: If `params['kwargs']` is None, won't this error out?

```
>>> a = {'test': 1}
>>> a.update(None)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: 'NoneType' object is not iterable
```

Meant to be `is not None`?

File: socs/agents/pysmurf_controller/agent.py [hunk 2]
```diff
@@ -767,7 +849,37 @@ def bias_dets(self, session, params):
...
+
+        Args
+        -------
+        disable_amps: bool
+            If True, will disable amplifier biases
+        disable_tones: bool
+            If True, will turn off RF tones and flux-ramp signal
+        """
+        if params['kwargs'] is None:
+            params['kwargs'] = {}
```
Comment (BrianJKoopman) [inline socs/agents/pysmurf_controller/agent.py:869]: Is this needed? Doesn't look like `params['kwargs']` is passed to anything later.

**Review by BrianJKoopman — CHANGES_REQUESTED** [review]
A couple comments inline below. Also, I know we talked about this offline, but the latest `develop` should be merged into this (or things rebased, though I don't know how that'll affect my comments.) That'll update the tests so they don't break. Then if you could write tests for the new tasks, that'd be awesome.

File: socs/agents/pysmurf_controller/agent.py (hunk 1 above)
Comment (jlashner) [reply socs/agents/pysmurf_controller/agent.py:785]: Oops good catch

File: socs/agents/pysmurf_controller/agent.py (hunk 2 above)
Comment (jlashner) [reply socs/agents/pysmurf_controller/agent.py:869]: Ah yeah this is probably a mistake

**Review by BrianJKoopman — APPROVED** [review]
Looks good, thanks for adding the new tests!

**Pull request merged**